### DISCUSSION TITLE
Welcome to wasm-bpf Discussions!

### DISCUSSION THREAD

**Discussion by yunwei37** [eunomia-bpf/wasm-bpf#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by XingZYu** [comment]
Hi, it seems there is a typo in the `README` of example `Bootstrap`. 
Example output:
```
$ sudo sudo ./wasm-bpf bootstrap.wasm -h
```
which should be 
```
$ sudo ./wasm-bpf bootstrap.wasm -h
```

**Reply by yunwei37** [reply]
Thanks for reporting! Would you mind opening a pull request for the project?
